spec/models/recipe_food_spec.rb

```diff
@@ -1,5 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe RecipeFood, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
```

Comment:
Would it make sense not to delete this file? It would need to be re-added in future PRs.